pkgs/by-name/zo/zoho-mail-desktop/package.nix

```
@@ -0,0 +1,36 @@
+{
+  lib,
+  appimageTools,
+  fetchurl,
+}:
+let
+  version = "1.6.5";
+  pname = "zoho-mail-desktop";
+
+  src = fetchurl {
+    url = "https://downloads.zohocdn.com/zmail-desktop/linux/${pname}-lite-x64-v${version}.AppImage";
+    hash = "sha256-TiPf5tQnVjPpzteSxIa4RLEyYBx6ohnnxPTAG0JjnrM=";
+  };
+
+  appimageContents = appimageTools.extractType1 { inherit pname version src; };
+in
+appimageTools.wrapType1 {
```

Likewise, wrapType1 is an alias of wrapType2
```suggestion
appimageTools.wrapType2 {
```